src/bacnet/basic/object/av.c

```diff
@@ -1438,6 +1438,341 @@ void Analog_Value_Intrinsic_Reporting(uint32_t object_instance)
 }
 
 #if defined(INTRINSIC_REPORTING)
+/**
```

Comment:
Not sure why these had not made to master already.... They're present in the av.h